mail-client/alot/alot-0.6.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 vcs-snapshot
```

Comment:
90% sure you don't need `vcs-snapshot` here.

Reply:
I tested to remove it, and it break the prepare phase so maybe there is a cleaner way but it seems needed, see log @mgorny 

```sh
 * Package:    mail-client/alot-0.6
 * Repository: local
 * Maintainer: aidecoe@gentoo.org
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux python_targets_python2_7 test userland_GNU
 * FEATURES:   network-sandbox nostrip preserve-libs sandbox test userpriv usersandbox
>>> Unpacking source...
>>> Unpacking alot-0.6.tar.gz to /var/tmp/portage/mail-client/alot-0.6/work
>>> Source unpacked in /var/tmp/portage/mail-client/alot-0.6/work
 * ERROR: mail-client/alot-0.6::local failed (prepare phase):
 *   The source directory '/var/tmp/portage/mail-client/alot-0.6/work/alot-0.6' doesn't exist
 *
 * Call stack:
 *            ebuild.sh, line  776:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1053:  Called __dyn_prepare
 *   phase-functions.sh, line  372:  Called die
 * The specific snippet of code:
 *              die "The source directory '${S}' doesn't exist"
 *
 * If you need support, post the output of `emerge --info '=mail-client/alot-0.6::local'`,
 * the complete build log and the output of `emerge -pqv '=mail-client/alot-0.6::local'`.
 * The complete build log is located at '/var/tmp/portage/mail-client/alot-0.6/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/mail-client/alot-0.6/temp/environment'.
 * Working directory: '/var/tmp/portage/mail-client/alot-0.6/homedir'
 * S: '/var/tmp/portage/mail-client/alot-0.6/work/alot-0.6'

 * Messages for package mail-client/alot-0.6:

 * ERROR: mail-client/alot-0.6::local failed (prepare phase):
 *   The source directory '/var/tmp/portage/mail-client/alot-0.6/work/alot-0.6' doesn't exist
 *
 * Call stack:
 *            ebuild.sh, line  776:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1053:  Called __dyn_prepare
 *   phase-functions.sh, line  372:  Called die
 * The specific snippet of code:
 *              die "The source directory '${S}' doesn't exist"
 *
 * If you need support, post the output of `emerge --info '=mail-client/alot-0.6::local'`,
 * the complete build log and the output of `emerge -pqv '=mail-client/alot-0.6::local'`.
 * The complete build log is located at '/var/tmp/portage/mail-client/alot-0.6/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/mail-client/alot-0.6/temp/environment'.
 * Working directory: '/var/tmp/portage/mail-client/alot-0.6/homedir'
 * S: '/var/tmp/portage/mail-client/alot-0.6/work/alot-0.6'
 *
 * The following package has failed to build, install, or execute postinst:
 *
 *  (mail-client/alot-0.6:0/0::local, ebuild scheduled for merge), Log file:
 *   '/var/tmp/portage/mail-client/alot-0.6/temp/build.log'
 *
```

And if we check the content of the work directory
```sh
$ ls /var/tmp/portage/mail-client/alot-0.6/work/
pazz-alot-fee5365
```